sdk/trace/batch_span_processor.go

```diff
@@ -378,6 +389,7 @@ func (bsp *batchSpanProcessor) enqueueBlockOnQueueFull(ctx context.Context, sd R
 		return true
 	default:
 		atomic.AddUint32(&bsp.dropped, 1)
+		bsp.o.DroppedSpansCallback()
```

Comment:
```suggestion
		go bsp.o.DroppedSpansCallback()
```

This should not allow the callback to block the processor.

Reply:
Thanks for the suggestion. Addressed in https://github.com/open-telemetry/opentelemetry-go/pull/2766/commits/7357a7f6d43e81067d30db228853d851c6aae263.